packages/e2e-tests/src/config/bootstrap.js

```diff
@@ -89,6 +89,13 @@ const ALLOWED_ERROR_MESSAGES = [
 
   // Another Firefox warning.
   'Layout was forced before the page was fully loaded',
+
+  // Expected css error.
+  'Stylesheet twentytwenty-block-editor-styles-css was not properly added.',
```

Comment:
Sounds like a bug that should reported in core.

Reply:
It seems to be a known issue. https://github.com/WordPress/gutenberg/pull/28165/files#r667483229

Reply:
Nice find! That explanation should be added as an inline comment.

I wouldn't consider an old comment on a merged PR to be a well-known issue though.

Do you know during which test this error is happening?

Given that `wp-admin/widgets.php` and `wp-admin/customize.php` don't show widgets for that theme, so it's not clear to me how this would happen.

If we know where this happens, this can be reported in core.

Reply:
Sorry, the comment isn't very good. It was just there for debugging. I will add a better comment.